swaggerpy/async_http_client.py

```diff
@@ -46,36 +46,21 @@ def start_request(self, request_params):
         """
         # request_params has mandatory: method, url, params, headers
         request_params = {
-            'method': str(request_params['method']),
+            'method': str(request_params.get('method', 'GET')),
```

Comment:
it looks unnecessary, as 'method' is a required field of operation. Validation done earlier would throw up.